src/util.py

```diff
@@ -146,18 +146,24 @@ def get_latest_gv_version(gv_major_version, channel):
         name += "-" + channel
     # A-C builds against geckoview-omni
     # See https://github.com/mozilla-mobile/android-components/commit/0b349f48c91a50bb7b4ffbf40c6c122ed18142d3
+    # However, geckoview-omni requires exoplayer2 which comes from the lite build, so check for that too
+    name_lite = name
     name += "-omni"
 
     r = requests.get(
         f"{MAVEN}/org/mozilla/geckoview/{name}/maven-metadata.xml"
     )
     r.raise_for_status()
     metadata = xmltodict.parse(r.text)
+    r = requests.get(
+        f"{MAVEN}/org/mozilla/geckoview/{name_lite}/maven-metadata.xml"
+    )
+    r.raise_for_status()
```

Comment:
Improvement for later (not in this PR): retry failed requests to be more resilient to network errors.

Reply:
The action runs every 15 minutes, so I don't think we need additional retries.